blosc/blosc2.c

```diff
@@ -973,11 +972,11 @@ static int blosc_c(struct thread_context* thread_context, int32_t bsize,
   blosc_timestamp_t last, current;
   float filter_time = 0.f;
 
+  // See whether we have a run here
   if (instr_codec) {
     blosc_set_timestamp(&last);
   }
 
-  // See whether we have a run here
```

Comment:
This is the correct place for the comment above.

Reply:
solved at f2fdffa71814dd1cbda6d3388e551c7e61214f64